packages/@okta/vuepress-site/docs/guides/sign-into-spa-redirect/main/vue/createproject.md

```diff
@@ -1,28 +1,23 @@
-You need recent versions of [Node](https://nodejs.org/en/) and [npm](https://www.npmjs.com/). Be sure to install the [Vue CLI](https://cli.vuejs.org/) as it is required for the steps below.
+You need recent versions of [Node](https://nodejs.org/en/) and [npm](https://www.npmjs.com/).
 
-1. Install Vue CLI with the following command:
+1. Create a Vue app using the following command:
 
    ```shell
-   npm install -g @vue/cli@4.5.15
+   npm create vue@3.11.2
    ```
 
-2. Create a Vue application named **okta-vue-quickstart** using the following command:
+1. You're asked to name the project and manually select features. Name the project `okta-vue-quickstart` and:
 
-   ```shell
-   vue create okta-vue-quickstart
-   ```
-
-3. You are asked to pick a present or manually select features. Choose `Manually select features`, and:
-
-* In the `Check the features needed for your project` menu, make sure that `Choose Vue version`, `Babel`, `Router`, and `Linter / Formatter` are selected using the spacebar.
-* Select **3.x** for the Vue.js version.
-* Select **Y** for router history mode.
-* Select the defaults for the other options.
+    * Select **No** for TypeScript.
+    * Select **No** for JSX Support.
+    * Select **Y** to add Vue Router for SPA development.
+    * Select the defaults for the other options.
 
-4. Go into your app's root app directory to view the created files:
+1. Go into your app's root app directory to view the created files and install dependencies:
 
    ```shell
    cd okta-vue-quickstart
+   npm install
    ```
 
-> **Note**: This sample application uses Vue CLI v4.5.15, okta-vue v5.1.1, and okta-auth-js v6.0.0.
+> **Note**: This sample app uses create-vue v3.11.2, okta-vue v5.7.0, and okta-auth-js v7.8.1.
```

Comment:
```suggestion
> **Note**: This sample application uses create-vue v3.11.2, okta-vue v5.7.0, and okta-auth-js v7.8.1.
```
Sorry for the inadvertent change

Reply:
This one is something our editing team is clamping down on. Is there a specific reason we need to spell out application? We're trying to be consistent across the docs